tests/plugins/filters.test.ts

```diff
@@ -218,6 +219,39 @@ describe("Filters plugin", () => {
       setCellContent(model, "A4", "Something");
       expect(zoneToXc(model.getters.getFilterTables(sheetId)[0].zone)).toEqual("A1:B3");
     });
+
+    test("Adjacent cells selection while applying filter on single cell", async () => {
+      const fixture = makeTestFixture();
```

Comment:
Do not do this. We are in a test file of a plugin. This has nothing related to UI and should not have something related to UI.
This line is use to make a test dom fixture, and is not useful in your test.
The same apply for `({ model } = await mountSpreadsheet(fixture, { model: new Model(data) }));`.
You can just instantiate your model with `const model = new Model(data);`.
If you need the UI for one of your test then your test should not belong to a plugin file test (but obviously it's not the case here).